angularjs-portal-home/src/main/webapp/my-app/layout/partials/home-header.html

```diff
@@ -13,16 +13,22 @@
       </div>
     </div>
 </div>
-<div class="inner-nav-container">
+<div class="inner-nav-container" ng-controller="LayoutController as layoutCtrl">
```

Comment:
We don't want to load this controller twice if we don't have to. Is there a way we could avoid this?